Crane/buildSrc/src/main/java/com/example/crane/buildsrc/Dependencies.kt

```diff
@@ -21,7 +21,7 @@ object Versions {
 }
 
 object Libs {
-    const val androidGradlePlugin = "com.android.tools.build:gradle:4.2.0-alpha08"
+    const val androidGradlePlugin = "com.android.tools.build:gradle:4.2.0-alpha09"
```

Comment:
alpha10 (ditto other projs)

Reply:
This is just for Jetsnack. Each sample is a separate PR.

Reply:
If they'll be done in separate PRs, remove all changes for the other samples?